test/lib/base-controller.js

```diff
@@ -41,6 +42,48 @@ describe('lib/base-controller', () => {
       });
     });
 
+    describe('.get()', () => {
+      const req = {};
+      const res = {
+        locals: {
+          partials: {
+            step: 'default-template'
+          }
+        }
+      };
+
+      beforeEach(() => {
+        res.render = sinon.stub();
+        hmpoFormWizard.Controller.prototype.get = sinon.stub();
+        controller = new Controller({
+          template: 'foo'
+        });
+      });
+
+      it('calls super', () => {
+        controller.get(req, res, _.noop);
+        hmpoFormWizard.Controller.prototype.get.should.have.been.calledOnce
+          .and.calledWithExactly(req, res, _.noop);
+      });
+
+      it('calls res.render with the template', () => {
+        controller.get(req, res, _.noop);
+        res.render.should.have.been.calledOnce;
+      });
+
+      it('sets template to res.locals.partials.step if view lookup error', () => {
+        res.render = (template, cb) => {
+          const err = {
+            message: 'Failed to lookup view'
+          };
+          cb(err);
```

Comment:
I'm not 100% sure, but the callback receives an instance of `Error`? If so, this should be `cb(new Error('Failed to lookup view'))`